FrameworkSystem/DB/ProxyDB.py

```diff
@@ -714,7 +714,7 @@ def __generateProxyFromProxyProvider(self, userDN, proxyProvider):
     if not result['OK']:
       return result
     pp = result['Value']
-    result = pp.getProxy({"DN": userDN})
+    result = pp.getProxy(userDN)
```

Comment:
The signature is inconsistent:

https://github.com/DIRACGrid/DIRAC/blob/3686c8b3cf297e19c388baa9a90edf9e9d65e47d/Resources/ProxyProvider/PUSPProxyProvider.py#L21
https://github.com/DIRACGrid/DIRAC/blob/3686c8b3cf297e19c388baa9a90edf9e9d65e47d/Resources/ProxyProvider/DIRACCAProxyProvider.py#L226

Reply:
now its the same